cabal.project

```diff
@@ -49,3 +49,8 @@ package network-mux
 
 package ouroboros-network
   flags: +asserts +cddl
+
+if impl(ghc >= 9.6)
+  allow-newer:
+    , *:base
+    , protolude:*
```

Comment:
We should create an issue in the protolude repo to support `ghc-9.6` (its `master` branch doesn't support it).

Reply:
There already is one.